### PR TITLE
[Feature-#5273][server-master] Task node of SWITCH

### PR DESCRIPTION
**Pull request by wangxj3**
[Feature-#5273][server-master] Task node of SWITCH
Determine the nodes that need to be run based on the value of the parameter.

### CODE REVIEW

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/TaskType.java [hunk 1]
```diff
@@ -51,7 +51,9 @@
     DATAX(10, "DATAX"),
     CONDITIONS(11, "CONDITIONS"),
     SQOOP(12, "SQOOP"),
-    WATERDROP(13, "WATERDROP");
+    WATERDROP(13, "WATERDROP"),
+    SWITCH(14, "SWITCH"),
```
Comment (Technoboy-) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/TaskType.java:55]: Do here really compile with no errors ?

**Review by Technoboy- — DISMISSED** [review]

**Review by brave-lee — COMMENTED** [review]
+1

**Review by lenboo — APPROVED** [review]
+1

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**